static/app/views/dashboardsV2/widgetBuilder/widgetLibrary/card.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from '@emotion/styled';
+
+import space from 'sentry/styles/space';
+import {WidgetTemplate} from 'sentry/views/dashboardsV2/widgetLibrary/data';
+import {getWidgetIcon} from 'sentry/views/dashboardsV2/widgetLibrary/widgetCard';
+
+type CardProps = {
+  iconColor: string;
+  onClick: () => void;
+  widget: WidgetTemplate;
+};
+
+export function Card({widget, iconColor, onClick}: CardProps) {
+  const {title, description, displayType} = widget;
+  const Icon = getWidgetIcon(displayType);
+
+  return (
+    <Container onClick={onClick}>
+      <IconWrapper backgroundColor={iconColor}>
+        <Icon style={{color: '#FFF'}} />
```

Comment:
```suggestion
        <Icon color="white" />
```

we can use the prop color 😉